packages/common/src/utils/arrays/ArrayUtils.test.ts

```diff
@@ -0,0 +1,75 @@
+import { ArrayUtils } from './ArrayUtils';
+
+describe('ArrayUtils', () => {
+  it('is defined', () => {
+    expect(ArrayUtils.ifTheSameElseDefault).toBeDefined();
+    expect(ArrayUtils.getUnitQuantity).toBeDefined();
+    expect(ArrayUtils.getSum).toBeDefined();
+    expect(ArrayUtils.immutablePatch).toBeDefined();
```

Comment:
Is this testing something not already captured by the type system?

Reply:
yeah it is, and that will be picked up in the pre-commit hook. tbh, I was being a little lazy and copied the existing pattern here, though these tests don't actually add anything very helpful. tests for tests sake.. and I think I'll just remove them